PubHub/PubHub.API/Controllers/BooksController.cs

```diff
@@ -161,7 +161,7 @@ public async Task<IResult> GetBookAsync(Guid id, [FromHeader] string appId)
         }
 
         [HttpPost("{id}/purchase")]
```

Comment:
Did we want to rename this to 'purchases' or should we just leave it?

Reply:
I csn rename it quick if that's desired 🤔🤷‍♂️

Reply:
Personally I don't really care. But if the standard says no "actions" in the URL(?) 🤔

Reply:
Yea. But we also broke the standard on users with `/suspend`

Reply:
Yeah, lets just move on then 🫡